### PR TITLE
Simplify node start-up

### PR DESCRIPTION
**Pull request by mrBliss**
Following https://github.com/input-output-hk/ouroboros-network/pull/938

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -57,7 +57,8 @@ extra-deps:
     #Ouroboros-network dependencies
 
   - git: https://github.com/input-output-hk/ouroboros-network
-    commit: d69ef321766b33a8d1152b7bfa039717b543f162
+    # TODO this commit points to a branch, never merge it!
```
Comment (mrBliss) [inline stack.yaml]: This is the only blocker (and updating the cabal and nix files)

File: cardano-node/src/Cardano/Node/Run.hs [hunk 2]
```diff
@@ -180,313 +152,99 @@ handleSimpleNode p NodeCLIArguments{..}
...
 
+    removeStaleLocalSocket myLocalSockPath
+
+    dbPath <- canonicalizePath =<< makeAbsolute ccDBPath
+
+    varTip <- atomically $ newTVar GenesisPoint
+
+    Node.run
+      (consensusTracers nodeTracers)
+      -- (withTip varTip $ chainDBTracer nodeTracers)
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Node/Run.hs]: Left over comment?

Comment (mrBliss) [reply cardano-node/src/Cardano/Node/Run.hs]: Good catch! The line below was tracing using `Show`, which we certainly wouldn't want.

**Pull request merged**